python/lsst/sims/maf/plots/spatialPlotters.py

```diff
@@ -145,15 +134,15 @@ def __call__(self, metricValue, slicer, userPlotDict, fignum=None):
         if False not in metricValue.mask:
             return None
         if plotDict['removeDipole']:
-            cl = hp.anafast(hp.remove_dipole(metricValue.filled(slicer.badval)))
+            cl = hp.anafast(hp.remove_dipole(metricValue.filled(slicer.badval)), lmax=plotDict['maxl'])
         else:
-            cl = hp.anafast(metricValue.filled(slicer.badval))
+            cl = hp.anafast(metricValue.filled(slicer.badval), lmax=plotDict['maxl'])
         l = np.arange(np.size(cl))
         # Plot the results.
         if plotDict['removeDipole']:
-            condition = ((l < plotDict['maxl']) & (l > 1))
+            condition = (l > 1)
```

Comment:
It's a minor thing but this is really hard to read since the 'l' looks almost the same as one.  Maybe use something like 'l_order'?